### PR TITLE
cleanup build

### PR DESCRIPTION
**Pull request by kyessenov**
/assign @ericvn

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

File: pilot/docker/Dockerfile.proxyv2 [hunk 1]
```diff
@@ -41,10 +41,5 @@ ENV ISTIO_META_ISTIO_VERSION $istio_version
 ARG TARGETARCH
 COPY ${TARGETARCH:-amd64}/pilot-agent /usr/local/bin/pilot-agent
 
-COPY stats-filter.wasm /etc/istio/extensions/stats-filter.wasm
```
[the remark this answers is not in the document]
Comment (kyessenov) [reply pilot/docker/Dockerfile.proxyv2:44]: helm charts ignore that field already AFAICT

[the remark this answers is not in the document]
Comment (howardjohn) [reply pilot/docker/Dockerfile.proxyv2:44]: ah cool, nvm then

**Review by ericvn — APPROVED** [review]
Thanks. I know there is already a doc update PR on removing these items there as well.

**Comment by kyessenov** [comment]
Yeah, docs need to be thoroughly updated to rewrite all EnvoyFilters to use
either Wasm or Telemetry APIs. I did the minimal work to not break it, but
we should do a followup as part of "disable EnvoyFilter by default".

**Pull request merged**